docs/source/release/v5.1.0/mantidworkbench.rst

```diff
@@ -5,172 +5,272 @@ Workbench Changes
+- Variables assigned in python scripts are now cleared when a script is run in its entirety.
+- When a running script is aborted, Mantid will automatically attempt to cancel the current algorithm rather than wait for it to complete. In some cases, this can result a lot of time saved!
 
 
 Bugfixes
+--------
+
+General
+#######
+
+- Workbench launch is no longer blocked when the Process ID assigned by project recovery has been reassigned to another programme.
+- Improved the handling of ``WorkspaceSingleValue`` workspaces in workbench. This fixes a crash which occurred when interacting with workspaces of this type.
+- Workbench will now spot if it is about to create the settings window off the available screen, and will move it so it is all visible. This is important as it is a modal dialog and could freeze the application in an unrecoverable way before.
+
+- Scripts no longer run slower if a plot is open.
+- The names of new tabs in the script editor now correctly incrementing on KDE display environments (i.e. KUbuntu).
+
+- Selecting spectra to plot that are out of range for the workspace is handled safely.
+- Opening the plot options for a sample logs plot now works.
+- Masking a workspace while its data table is displayed now works as expected.
+- There is now a warning if ``File > Generate Recovery Script`` is executed with no workspaces present.
+
+Plotting
 ########
 
-- Fixed new tab names not incrementing correctly on KDE display environments (i.e. KUbuntu).
-- Fixed a bug where setting columns to Y error in table workspaces wasn't working. The links between the Y error and Y columns weren't being set up properly.
-- Fixed a crash when you selected a spectra to plot that was not present in a workspace.
-- Fixed a crash when opening the plot options for a sample logs plot.
-- Fixed a crash when you defined a new Fit Function after deleting a plot.
-- Fixed a crash when plotting the logs from a multi-dimensional workspace, that combines several different original workspaces.
-- Fixed a crash when masking a workspace while the worspace data table was on the screen.
+- ``plt.show()`` now shows the most recently created figure.
+- The logs from a multi-dimensional workspace (that combines several different original workspaces) can be successfully plotted.
+- Opening figure options on a plot with an empty legend now works.
+- The plot setting for default axis scale now applies to the correct axis.
+- Overplotting no longer resets the axes scales.
+- Altering a line/curve no longer changes the order of curves in the legend.
+- The double-click axis menus now work as expected for tiled plots.
+- 3D Surface plots of data with monitors now have the correct initial colorbar limits.
+- The plot toolbar now shows the correct toolbar buttons for 3D plots.
+- Reduce log level of project recovery when it fails to save a plot correctly, from warning to debug, to avoid excess messages in the console.
+
+Colorfill Plots
+^^^^^^^^^^^^^^^
```

Comment:
```suggestion
###############
```

Reply:
I'd like to keep this one a sub-sub section of Bugfixes>Plotting>